packages/neon-core/src/u/Fixed8.ts

```diff
@@ -1,6 +1,19 @@
 import BN from "bignumber.js";
 import { reverseHex } from "./misc";
 
+const DECIMALS = 100000000;
+
+// 0x7fffffffffffffff = 9223372036854775807 (maximum hex fixed8)
+const MAX_FIXED8_HEX = new BN(2).pow(63).minus(1);
+export const MAX_FIXED8 = MAX_FIXED8_HEX.div(DECIMALS);
+
+// 0x8000000000000000 = -9223372036854775808 (minimum hex fixed8)
+const MIN_FIXED8_HEX = new BN(2).pow(63).negated();
+export const MIN_FIXED8 = MIN_FIXED8_HEX.div(DECIMALS);

```

Comment:
How about changing this to a static readonly property Fixed8.MIN_VALUE and Fixed8.MAX_VALUE? That is similar to Number.MIN_VALUE and Number.MAX_VALUE in js.